### DISCUSSION TITLE
[Question]: Procedure to update to 0.14 with Home Assistant Addon for Frigate (For old x86_64 CPU without AVX)

### DISCUSSION THREAD

**Discussion by BigRedHolger** [blakeblackshear/frigate#13170] [Ask A Question] [answered]
### What is your question:

I am unsure how to update. I have come to believe that I must uninstall and reinstall the Frigate Addon to update. Is that correct?

**Comment by NickM-27** [comment, marked as answer]
You'll have to wait for the addon PR to be merged

**Reply by BigRedHolger** [reply]
Thanks, NickM-27! I don't suppose you have any info on where I might find the timeline of the merger?
